Makefile

```diff
@@ -134,9 +136,11 @@ build-stack-package: $(STACK_PACKAGE_REGISTRY)
 	@$(INFO) building stack package in $(STACK_PACKAGE)
 	@find $(CRD_DIR) -type f -name '*.yaml' | \
 		while read filename ; do cat $$filename > \
-		$(STACK_PACKAGE_REGISTRY)/resources/$$( basename $${filename/.yaml/.crd.yaml} ) \
+		$(STACK_PACKAGE_REGISTRY_RESOURCES)/$$( basename $${filename/.yaml/.crd.yaml} ) \
 		; done
 	@cp -r $(STACK_PACKAGE_REGISTRY_SOURCE)/* $(STACK_PACKAGE_REGISTRY)
+	@cp -r $(STACK_PACKAGE_REGISTRY_SOURCE_RESOURCES)/* $(STACK_PACKAGE_REGISTRY_RESOURCES)
```

Comment:
I've opened https://github.com/crossplaneio/crossplane-cli/issues/12 to track adding this to `stack init` after v0.3.

The WordPress stack will be updated for v0.3 in a similar fashion to the cloud provider stacks so that it is a complete (rich) stack experience.

Reply:
I initially thought that `..._RESOURCES` represented a subdirectory and so I thought the added `cp -r` command may have been unnecessary.

Based on the current usage, it may be less confusing to name these:
* `STACK_PACKAGE_REGISTRY_MANIFESTS` 
* `STACK_PACKAGE_REGISTRY_RESOURCES`

Reply:
Why aren't we just using `config/stack/manifests/resources` for these files?
We wouldn't need `Makefile` changes.

Reply:
that's a very good point @displague! i've made the update you suggested to include the `resources` dir under `config/stack/manifests` and no further changes are needed to the makefile right now :)